WrightTools/kit.py

```diff
@@ -637,7 +644,7 @@ def read_headers(filepath):
 
 
 def write_h5(filepath, dictionary):
-    """ Save a python dictionary into an `HDF5 <https://www.hdfgroup.org/HDF5/doc/H5.intro.html>`_
+    """ Save a python dictionary into an `HDF5`_
```

Comment:
in this case you're the expert @ksunden---whatever you think is the best strategy